test/fixture.rb

```diff
@@ -89,14 +97,14 @@ def after_b(return_value, array)
 
     class BeforeAfterWithStateC
       # :c
-      def before_c(string:)
+      def before_c(a, string:)
```

Comment:
what is a ?

Reply:
ignore the minitests, those are old tests that i'll delete